lib/rules/template-indent.js

```diff
@@ -0,0 +1,200 @@
+const { builtinRules } = require('eslint/use-at-your-own-risk');
+
+const baseRule = builtinRules.get('indent');
+const IGNORED_ELEMENTS = new Set(['pre', 'script', 'style', 'textarea']);
+
+const schema = baseRule.meta.schema.map((s) => ({ ...s }));
+schema[1].properties = {
+  ignoredNodes: schema[1].properties.ignoredNodes,
+  ignoreComments: schema[1].properties.ignoreComments,
+};
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  ERROR_MESSAGE: baseRule.meta.messages.wrongIndentation,
```

Comment:
Let's avoid exporting `ERROR_MESSAGE`. That isn't necessary anymore now that we have `meta.messages`.